examples/minimal/rendered-config.yml

```diff
@@ -0,0 +1,42 @@
+"APPS_DMS_DEFAULT_CLOUD_PROVIDER": "s3"
```

Comment:
No need to check in the rendered config. Seems worthwhile to add to the .gitignore file.

Reply:
[Deleted](https://github.com/franco-caylent/terraform-aws-tamr-config/commit/16e42000a8edb4c853644446e1be70d1e6a4b02a) and [added to gitignore](https://github.com/franco-caylent/terraform-aws-tamr-config/commit/4cf61e31dce5d04af4641b75b62f0eba87401a65)